streams/in.m3u

```diff
@@ -1,6 +1,4 @@
 #EXTM3U
-#EXTINF:-1 tvg-id="",3 Tamil TV (720p) [Not 24/7]
-https://6n3yogbnd9ok-hls-live.5centscdn.com/threetamil/d0dbe915091d400bd8ee7f27f0791303.sdp/index.m3u8
```

Comment:
works totally fine for me. Does it need a removal?
![image](https://github.com/iptv-org/iptv/assets/5430116/36e38285-d524-4b51-b516-13f147ed5106)

Reply:
[I moved it here.](https://github.com/iptv-org/iptv/pull/14997/files#diff-17ca22b929d4575883d9d9936826bcaaa700f0eb18bf9ae0eaa0c4ef52d5178aR6-R7)